### ISSUE TITLE
While typing in the address bar, it's impossible to explore its content with a screen reader

### ISSUE THREAD

**Issue by BlindMoon38** [mozilla-mobile/fenix#20491]
### Steps to reproduce

1. Open Firefox with TalkBack enabled.
2. Tap the address bar and start typing.
3. Try to explore the typed text via TalkBack navigation gestures (characters, words, etc.).

### Expected behaviour

* While typing, TalkBack should repeat the characters as they're being typed,.
* After typing a word followed by a space, TalkBack should read the typed word.
* one should be able to explore typed text as well as URLs using TalkBack navigation gestures (characters, words, etc.).

### Actual behaviour

* Character echo while typing doesn't work.
* Word echo while typing doesn't work.
* It's impossible to explore the actual Tex in the address bar with TalkBack gestures.

### Device name

Xiaomi Redmi Note 8

### Android version

MIUI 12.0.4.0, Android 10

### Firefox release type

Firefox Nightly

### Firefox version

92.0a1 (Build #2015823243) AC: 92.0.20210719094322, 97d95e60fb GV: 92.0a1-20210718090829 AS: 79.0.0

### Additional information

TalkBack version 9.1

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-17351)

**Comment by DiegoARL38** [comment]
This issue is still present, a year and four months later... I know these things take time, but this is a major accessibility bug in a crucial part of the app...

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1811600

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
